src/App.tsx

```diff
@@ -1,24 +1,182 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, {
+  useEffect,
+  useState,
+  ChangeEvent,
+  FormEvent,
+} from 'react';
 
-const USER_ID = 0;
+import { Header } from './components/Header';
+import { Main } from './components/Main';
+import { Footer } from './components/Footer';
+import { ErrorPanel } from './components/ErrorPanel';
+
+import { Todo } from './types/Todo';
+import { Filters } from './types/Filters';
+
+import { getTodos, addTodo, removeTodo } from './api/todos';
+
+const USER_ID = 6160;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [isEditing] = useState(false);
+  // const [hasChangesInTodos, setHasChangesInTodos] = useState(0);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>(todos);
+  const [selectedFilter, setSelectedFilter] = useState<Filters>(Filters.All);
+  const [todoTitleToAdd, setTodoTitleToAdd] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [idsOfLoadingTodos, setIdsOfLoadingTodos] = useState<number[]>([]);
```

Comment:
what is this state for? Probably I can't get it because of the name

Reply:
ye i had hard times adding a name for this. 
its for storing the ID-s of the todos which I want to move to a loading state on clicking the 'Clear completed' button.
or a single ID of a todo for the loading when removing it from the list.

Reply:
do you really need to store them in the state? It does not seem to affect the page behavior

Reply:
![image](https://user-images.githubusercontent.com/109879480/217327527-1960cb98-9fcd-4b3e-bc5a-39de09d519b1.png)
i push them into the array, then clear the array.
I think i need them. unless i figure out a better way to track them for the multiple loading states.

Reply:
![image](https://user-images.githubusercontent.com/109879480/217328334-d097a59b-db6a-4aa9-b592-9ab576196944.png)